GA-worksheet.md

```diff
@@ -7,61 +7,78 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+ The header tag of a website holds other tags inside it like <body>.
```

Comment:
The header tag is actually within the body. It holds the introductory material for the website, such as a banner, logo, or navigational links. Check it out on Buzzfeed.com. The top of the page that lets you know that it is  Buzzfeed is the header (inspect element to clarify).